src/main/java/Storage.java

```diff
@@ -0,0 +1,120 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.io.File;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+
+/**
+ * Saves and loads files when Duke starts and closes
+ */
+public class Storage {
+
+    /**
+     * Location of saved files
+     */
+    private String filePath;
+
+    /**
+     * Creates a Storage object that can save and load tasks
+     *
+     * @param filePath Location of saved files
+     */
+    public Storage (String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads previously saved files into Duke when Duke starts up
+     *
+     * @return An Arraylist of task that contains saved files
+     * @throws IOException when reader takes in NULL input
+     */
+    public ArrayList<Task> loadFiles() throws IOException {
+        ArrayList<Task> lst = new ArrayList<>();
```

Comment:
A small little note but I think the standard naming convention for collections are plural. Eg. lsts instead of lst.